test/cmd/help.sh

```diff
@@ -50,6 +50,7 @@ os::cmd::expect_success_and_text 'oadm' 'Install Commands:'
 os::cmd::expect_success_and_text 'oadm ca' 'Manage certificates'
 os::cmd::expect_success_and_text 'openshift start kubernetes' 'Kubernetes server components'
 os::cmd::expect_success_and_text 'oc exec --help' '\[options\] \-\- COMMAND'
+os::cmd::expect_success_and_text 'oc rsh --help' '\[options\] \[COMMAND\]'
```

Comment:
Is there a reasonable way to walk the usage from our upstream commands to see if they match?  Similar to how we have reflective unit tests to check for parity.